pkg/api/types.go

```diff
@@ -1238,6 +1238,8 @@ const (
 	ResourceMemory ResourceName = "memory"
 	// Volume size, in bytes (e,g. 5Gi = 5GiB = 5 * 1024 * 1024 * 1024)
 	ResourceStorage ResourceName = "storage"
+	// Number of Pods that may be running on this Node.
+	ResourceMaxPods ResourceName = "maxpods"
```

Comment:
`maxpods` is not an acceptable resource name. Note that none of the other resources have "max" in their names. Whether a resource is a "max" or "min" depends on whether it is in the `limits` map or the `requests` map. If we were to have such a resource, it should simply be named `pods`.